### ISSUE TITLE
not supported on 2017-02-19b "Frusterick Manners"?

### ISSUE THREAD

**Issue by jkldgoefgkljefogeg** [Taggic/orphanmedia#18]
After installing through Extension Manager,

> Warning: Declaration of syntax_plugin_orphanmedia::handle($match, $state, $pos, Doku_Handler &$handler) should be compatible with DokuWiki_Syntax_Plugin::handle($match, $state, $pos, Doku_Handler $handler) in /opt/wwwroot/dokuwiki/lib/plugins/orphanmedia/syntax.php on line 839
> 
> Warning: Declaration of syntax_plugin_orphanmedia::render($format, Doku_Renderer &$renderer, $data) should be compatible with DokuWiki_Syntax_Plugin::render($format, Doku_Renderer $renderer, $data) in /opt/wwwroot/dokuwiki/lib/plugins/orphanmedia/syntax.php on line 839
> This tab gives you access to all available 3rd party plugins and templates for DokuWiki. Please be aware that installing 3rd party code may pose a security risk, you may want to read about plugin security first.

?id=orphanmedia:example returns This topic does not exist yet

tested on Release 2017-02-19b "Frusterick Manners"

**Comment by Taggic** [comment]
right, dokuwiki changed the function and assumes all plugins will Change use of built in function towards that. unfortunately i've currently no time for code conversion towards dokuwiki changes. sorry for that :-(

at least it is just a warning. this plugin should be used anyway only on pages wher just admin has Access to. reason is the Performance consumption while running this plugin will reduce Performance for all other users.

finally this plugin is not actively maintained anymore.
